README.md

```diff
@@ -29,6 +31,10 @@ There has to be a JSON-file with descriptions of the projects you want to deploy
 }
 ```
 
+### Deploy chewie with chewie
+If you want to deploy chewie on a server with another hostname than `<server>.abakus.no` the 
+`CORRECT_HOSTNAME` in order to run `make production`. Default is `abakus.no`
```

Comment:
If you want to deploy chewie on a server with another hostname than `<server>.abakus.no` **expose** the `CORRECT_HOSTNAME` **environment variable** in order to run `make production`.